src/pages/[platform]/build-a-backend/storage/configure-access/index.mdx

```diff
@@ -54,6 +54,12 @@ import common_configureaccess from '/src/fragments/lib/storage/native_common/con
 
 <InlineFilter filters={['javascript',  'react-native', 'angular', 'nextjs', 'react', 'vue']}>
 
+<Callout warning>
+We recommend using the new [Amplify Gen2](https://docs.amplify.aws/gen2/build-a-backend/storage/) experience when defining file access permissions as it offers a more flexible approach to customize access to files.
+
+Note: `accessLevel` parameter is deprecated and maybe removed in next major version.
```

Comment:
Should we include both params?

```suggestion
Note: The `key`, `accessLevel`, and `targetIdentityId` parameters are deprecated and maybe removed in next major version.
```

Reply:
since this section is specific to accessLevel we had it like this. Would u still prefer keeping it consistent?

Reply:
MSTM!